### PR TITLE
LRDOCS-3791 Sync docs updates

### PR DESCRIPTION
**Pull request by ngaskill**
Added note about HTTPS security certificates to article on desktop Sync. Also changed Sync docs to use product tags instead of "Liferay" mentions.

### CODE REVIEW

File: discover/portal/articles/03-collaboration/01-managing-documents-and-media/06-using-liferay-sync-desktop.markdown [hunk 1]
```diff
@@ -65,16 +65,27 @@ Next, you'll configure the Sync client.
...
-   *Search* bar above the site list. If you want to sync all the subfolders of 
-   your selected sites, click *Proceed* and move on to the next step. 
+    When connecting to a server via HTTPS, an error appears if the certificate 
+    can't be verified. Choosing *Proceed Anyway* bypasses verification and 
+    leaves the connection open to compromise. Liferay Sync attempts to read the 
+    certificates specified in the Java Control Panel 
+    ([see section 20.4.5](https://docs.oracle.com/javase/8/docs/technotes/guides/deploy/jcp.html#A1152831)). 
+    If Java isn't installed, you can also put your certificates in 
+    [user.home]/.liferay-sync-3/certificates. Liferay Sync will trust all 
+    certificates.
```
Comment (dejuknow) [inline discover/portal/articles/03-collaboration/01-managing-documents-and-media/06-using-liferay-sync-desktop.markdown]: Is this sentence correct? "Liferay Sync will trust all certificates"?

Should it be: "Liferay Sync will trust all certificates in this folder"?

Comment (ngaskill) [inline discover/portal/articles/03-collaboration/01-managing-documents-and-media/06-using-liferay-sync-desktop.markdown]: I'm not sure. I took that sentence directly from the ticket, assuming it to be true.

Comment (dejuknow) [inline discover/portal/articles/03-collaboration/01-managing-documents-and-media/06-using-liferay-sync-desktop.markdown]: oops. My bad. I didn't complete the sentence in the ticket description. Can we change it to "Liferay Sync will trust all certificates in this folder."

Comment (ngaskill) [inline discover/portal/articles/03-collaboration/01-managing-documents-and-media/06-using-liferay-sync-desktop.markdown]: No problem. I'll push up the change.

**Pull request merged**